test/testinput/bufr_ncep_esamua.yaml

```diff
@@ -70,16 +77,17 @@ observations:
                 _3: metop-b
                 _4: metop-a
                 _5: metop-c
-#               _206: noaa-15
-                _209: noaa-18
-                _223: noaa-19
+#               _206: n15
+                _209: n18
+                _223: n19
```

Comment:
either NOAA16 or NOAA17 lasted like a year correct?  
should this just list all the possible ones?